README.md

```diff
@@ -1 +1,20 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] Your repo should have ```.gitignore``` file, please include it.
-  ```test.md``` and ```node_modules``` folder should be ignored by git

Reply:
I also added the .gitignore file to my branch.